user/advanced-topics/standalones-and-hvms.md

```diff
@@ -44,7 +44,7 @@ You can create a standalone in the Qube Manager by selecting the "Type" of "Stan
 Alternatively, from the dom0 command line:
 
 ```
-qvm-create --class StandaloneVM --label <label> --property virt_mode=hvm <vmname>
+qvm-create --class StandaloneVM --label <LABEL> --property virt_mode=hvm <VMNAME>
```

Comment:
Out of curiosity, why `<VMNAME>` here but `<QUBE>` below?

Reply:
Ah, guessing it was just a capitalization change of the existing words without further thought given to changing them.

Reply:
No idea. I was in auto-pilot -- just made it caps everywhere. Probably `<QUBE>` would be better. Feel free to change it.

Reply:
No worries. Thank you.